src/Tickets/Seating/app/blockEditor/store/common-store-bridge.js

```diff
@@ -31,4 +37,15 @@ export function setCappedTicketCapacityInCommonStore(clientId, capacity) {
 	dispatchToCommonStore(setTicketTempCapacity(clientId, capacity));
 	dispatchToCommonStore(setTicketCapacityType(clientId, CAPPED));
 	dispatchToCommonStore(setTicketTempCapacityType(clientId, CAPPED));
+	setTicketHasChangesInCommonStore(clientId);
+}
+
+export function setTicketHasChangesInCommonStore(clientId) {
+	const ticketTempTitle = selectFromCommonStore(getTicketTempTitle, {
+		clientId,
+	});
+
+	// "Changing" the tempTitle will trigger a re-evaluation on the confirm button's disabled status.
```

Comment:
This comments gives me pause: it makes the method look like a "hack" of how the existing block works.

If I understand this correctly, the confirm button disabled status will **not** be re-evalutated on the call to `setTicketHasChanges`, right?
This behaviour, while it might be pre-existing, does not seem correct. If a filter has been introduced to filter the ticket valid/invalid status, then a change to that status should be all that is required to trigger a re-evaluation of the confirm button disabled status.
The fact that `setTicketHasChanges`, as an API, will not work as intended and will require a call to `setTicketTempTitle` to work is not documented and will eventually break sending the next dev down a rabbit hole of debugging.

If `setTicketHasChanges` is not enough to trigger re-evaluation, this is the moment where that should be changed in the original reducers.

Am I missing something?

Reply:
I have retested this and it seems it's not needed to trigger fake changes to tempTitle to re-evaluate the button's disabled status. Triggering that it has changes is enough. I am not sure as to why initially i though it was needed, which confuses me a bit.